config/initializers/jsonp.rb

```diff
@@ -0,0 +1,80 @@
+require 'rack'
```

Comment:
+1. I would rather not install the gem, specifically because 1. it is dead and 2. the code isn't very complex. @DeeDeeG made a suggestion below to use `rack-cors` which hopefully will be a better alternative